### DISCUSSION TITLE
LibPNG Error: Bad Adaptive Filter Value

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#497] [Q&A] [answered]
what
![image](https://github.com/user-attachments/assets/65794a4c-9205-4a5a-8f3e-c4c715a02d9d)
for context, update.bat isn't working for me so I manually installed all of the libraries

**Comment by SenDoesStuff** [comment]
Alot of this could be answered in the codename engine discord help channel btw

**Reply by THEkatinamicrowave** [reply]
my dad blocked discord on my computer because unrelated incident

**Reply by THEkatinamicrowave** [reply]
know what screw this I'm deleting all of the .haxelibs and rerunning update.bat

**Reply by THEkatinamicrowave** [reply, marked as answer]
upon further notice, I've discovered libs.xml
